src/cloud/schemas/ProvisionUserDeleteRequest.yml

```diff
@@ -0,0 +1,26 @@
+type: object
+properties:
+  userID:
+    type: string
+    description: user id to delete
+  orgID:
+    type: string
+    description: organization id the user should be deleted from
+  newOwnerID:
+    type: string
+    description: user id of the owner that user-owned resources should be transferred to, if transferring user-owned resources
+  tokenOption:
+    type: string
+    enum:
+      - transfer
+      - delete
+    description: organization id the user should be deleted from
```

Comment:
the description is not for `tokenOption`.

Reply:
It sure isn't...fixed!